src/DevBetterWeb.Core/Services/MemberSubscriptionPeriodCalculationsService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+
+namespace DevBetterWeb.Core.Services
+{
+  public class MemberSubscriptionPeriodCalculationsService : IMemberSubscriptionPeriodCalculationsService
+  {
+    private const int DAYS_SUBSCRIBED_TO_BECOME_ALUMNUS = 730;
+
+    // none of these methods should ever be called if member does not have current subscription
```

Comment:
How do you enforce this design intent? Add a Guard clause to check if they have no subscriptions and throw, for instance. Once you have those, you no longer need this comment.